plugins/mkt-queue-sync/src/classes/Synchronizer.php

```diff
@@ -0,0 +1,269 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MikrotikQueueSync;
+
+use Ubnt\UcrmPluginSdk\Security\PermissionNames;
+use Ubnt\UcrmPluginSdk\Service\UcrmApi;
+use Ubnt\UcrmPluginSdk\Service\UcrmOptionsManager;
+use Ubnt\UcrmPluginSdk\Service\PluginConfigManager;
+use Ubnt\UcrmPluginSdk\Service\PluginLogManager;
+//use Ubnt\UcrmPluginSdk\Service\UcrmSecurity;
```

Comment:
Just remove not used code instead of just make them comment with `//`. (eg. this four `use` statements)
 
```suggestion
```

Reply:
Hi @janprochazkacz i think now all changes you asked for are made.... please let me know if you see anything elso to correct...

In other hand, maybe can you help me to update my plugin.json file so travis can give the OK? i already tried to run php generate-json.php > plugins.json but i get an error saying "stdout is not a tty"